config/config-api/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>uk.com.robust-it</groupId>
             <artifactId>cloning</artifactId>
-            <version>1.7.3</version>
+            <version>1.9.2</version>
```

Comment:
you might want to double check. but i have seen clone failures for newer releases of this lib.